modules/nf-core/gmmdemux/main.nf

```diff
@@ -0,0 +1,71 @@
+
+process GMMDEMUX {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gmm-demux:0.2.2.3--pyh7cba7a3_0':
+        'biocontainers/gmm-demux:0.2.2.3--pyh7cba7a3_0' }"
+
+    input:
+    tuple val(meta), path(cell_hashing_barcodes,stageAs: "hto_files/*"), path(cell_hashing_matrix,stageAs: "hto_files/*"),path(cell_hashing_features,stageAs: "hto_files/*"),val(hto_names)
+    val csv
+    val num_cells
+    val output_dir
+    val full_report
+    val simplified_report
+    val examine
+    val ambigous
+    val extract
+    val skip
```

Comment:
These don't [appear to to be mandatory](https://github.com/CHPGenetics/GMM-Demux?tab=readme-ov-file#case-1-basic-usage-remove-msms), in which case they should all be handled by [`ext.args` not as input channels](https://nf-co.re/docs/guidelines/components/modules#optional-command-arguments).

Reply:
I think that is up to the pipeline developer to configure and check if it is working within the pipeline or not (as in they should specify the 'allowed' tool parameters in the pipeline's modules.config file, to ensure the pipeline works as they want it to - the module doesn't necessarily need to check everythin) 

I would still remove anything that isn't mandatory